pkg/kf/testutil/integration.go

```diff
@@ -571,3 +571,45 @@ func (k *Kf) Doctor(ctx context.Context) {
 	PanicOnError(ctx, k.t, "doctor", errs)
 	StreamOutput(ctx, k.t, output)
 }
+
+// UploadBuildpacks runs the UploadBuildpacks command.
+func (k *Kf) UploadBuildpacks(ctx context.Context, flags map[string]string) {
+	k.t.Helper()
+	Logf(k.t, "running upload-buildpacks...")
+	defer Logf(k.t, "done running upload-buildpacks.")
+
+	args := []string{
+		"upload-buildpacks",
+		"--built-in",
+	}
+
+	for k, v := range flags {
+		if v == "" {
+			args = append(args, k)
+			continue
+		}
+
+		args = append(args, fmt.Sprintf("%s=%s", k, v))
```

Comment:
Why is this necessary over appending the value alone?

Reply:
I was attempting better readability, but I think variadic strings will actually do better.